### ISSUE TITLE
Maya Tool - Solver UI - Setting Static + Animated and Per-Frame strategy triggers an error

### ISSUE THREAD

**Issue by david-cattermole** [david-cattermole/mayaMatchMoveSolver#43]
Using the Solver UI, and setting a Solver Step to:
- Attributes = "Static + Animated"
- Strategy = "Per-Frame"

This combination causes an traceback.

This traceback is correct, static attributes cannot be solved for using per-frame strategy.
However, setting this combination should not be possible.
We must make sure Per-Frame is not a valid option, in the Solver UI.

Full Traceback:
```
# Traceback (most recent call last):
#   File "/home/davidc/maya/2017/modules/mayaMatchMoveSolver-0.2.0-maya2017-linux/python/mmSolver/ui/uimodels.py", line 472, in setData
#     set_attr_func(value)
#   File "/home/davidc/maya/2017/modules/mayaMatchMoveSolver-0.2.0-maya2017-linux/python/mmSolver/tools/solver/ui/solver_nodes.py", line 180, in setStrategy
#     self.setStepNode(n)
#   File "/home/davidc/maya/2017/modules/mayaMatchMoveSolver-0.2.0-maya2017-linux/python/mmSolver/tools/solver/ui/solver_nodes.py", line 117, in setStepNode
#     lib_collection.set_named_solver_step_to_collection(col, node)
#   File "/home/davidc/maya/2017/modules/mayaMatchMoveSolver-0.2.0-maya2017-linux/python/mmSolver/tools/solver/lib/collection.py", line 315, in set_named_solver_step_to_collection
#     set_solver_step_list_to_collection(col, step_list)
#   File "/home/davidc/maya/2017/modules/mayaMatchMoveSolver-0.2.0-maya2017-linux/python/mmSolver/tools/solver/lib/collection.py", line 351, in set_solver_step_list_to_collection
#     sol_list = compile_solvers_from_steps(col, step_list)
#   File "/home/davidc/maya/2017/modules/mayaMatchMoveSolver-0.2.0-maya2017-linux/python/mmSolver/tools/solver/lib/collection.py", line 363, in compile_solvers_from_steps
#     tmp_list = step.compile(col, override_current_frame=use_cur_frame)
#   File "/home/davidc/maya/2017/modules/mayaMatchMoveSolver-0.2.0-maya2017-linux/python/mmSolver/tools/solver/lib/solver_step.py", line 212, in compile
#     strategy,
#   File "/home/davidc/maya/2017/modules/mayaMatchMoveSolver-0.2.0-maya2017-linux/python/mmSolver/tools/solver/lib/solver_step.py", line 80, in _solve_all_attrs
#     raise ValueError(msg % strategy)
# ValueError: strategy is not supported: strategy='per_frame'
```

**Comment by david-cattermole** [comment]
Another example of the traceback error:
```
# Error: ValueError: file C:\Users\Anil Reddy\Documents\maya\2019\modules\mayaMatchMoveSolver-0.2.0-maya2019-win64\python\mmSolver\tools\solver\lib\solver_step.py line 80: strategy is not supported: strategy=u'per_frame'
```

**Comment by david-cattermole** [comment]
This has been addressed in commit 4ae01a88dd8c3ce7efc573255f039c6add1fbaf6.
